Ble rssi svc (#377)
OttoWinter/esphomedocs: components/binary_sensor/ble_presence.rst

      scan_interval: 300s

    binary_sensor:
      # Presence based on MAC address
      - platform: ble_presence
        mac_address: AC:37:43:77:5F:4C
        name: "ESP32 BLE Tracker Google Home Mini"
      # Presence based on BLE Service UUID
      - platform: ble_presence
        service_uuid: '11aa'
        name: "ESP32 BLE Tracker Test Service 16 bit"

.. note::

    Service UUID can be 16 bit long, as in the example, but it can also be 32 bit long

>     like `1122aaff`, or 128 bit long like `11223344-5566-7788-99aa-bbccddeeff00`.

Reviewer: ```suggestion
    like ``1122aaff``, or 128 bit long like ``11223344-5566-7788-99aa-bbccddeeff00``.
```

Same here, this is RST syntax.

Configuration variables:
------------------------

-  **name** (**Required**, string): The name of the binary sensor.
-  **mac_address** (*Optional*, MAC Address): The MAC address to track for this

>    binary sensor. Either this or ''service_uuid'' has to be present.

Reviewer: ```suggestion
   binary sensor. Either this or ``service_uuid`` has to be present.
```

-  **service_uuid** (*Optional*, string) 16 bit, 32 bit, or 128 bit BLE Service UUID
   which can be tracked if the device randomizes the MAC address. Either

>    this or ''mac_address'' has to be present.

Reviewer: ```suggestion
   this or ``mac_address`` has to be present.
```

      - platform: ble_presence
        mac_address: AC:37:43:77:5F:4C
        name: "ESP32 BLE Tracker Google Home Mini"

>       - platform: ble_presence

Reviewer: 1. Please only add one example for service_uuid here. The example at the top should always be minimal to give the user an idea. You can create a `.. note::`` below with more examples with different uuid lengths.

2. Please put a comment above each of these two entries (mac and uuid). One saying `# Presence based on MAC address` and the other the same but with UUID.

        service_uuid: '11aa'
        name: "ESP32 BLE Tracker Test Service 16 bit"
      - platform: ble_presence
        service_uuid: '11aa11aa'
        name: "ESP32 BLE Tracker Test Service 32 bit"
      - platform: ble_presence
        service_uuid: '11223344-5566-7788-99aa-bbccddeeff00'
        name: "ESP32 BLE Tracker Test Service 128 bit"

Configuration variables:
------------------------

-  **name** (**Required**, string): The name of the binary sensor.
-  **mac_address** (*Optional*, MAC Address): The MAC address to track for this

>    binary sensor. Either this or **service_uuid** has to be present.

Reviewer: ```suggestion
   binary sensor. Either this or ``service_uuid`` has to be present.
```

> -  **service_uuid** (*Optional*, 16 bit, 32 bit, or 128 bit BLE Service UUID): The BLE

Reviewer: ```suggestion
-  **service_uuid** (*Optional*, string): The BLE
```

Usually, the second parameter here shows what YAML type it is (though that seems to not have been enforced with MAC address, feel free to change that to `string` too).

   Service UUID which can be tracked if the device randomizes the MAC address. Either

>    this or **mac_address** has to be present.

Reviewer: ```suggestion
   this or ``mac_address`` has to be present.
```

currently not supported.

Please note that devices that show a ``RANDOM`` address type in the logs cannot be used for
MAC address based tracking, since their MAC-address periodically changes. Instead you can create
a BLE beacon, set a unique 16 bit, 32 bit or 128 bit Service UUID and track your device based on that.

> Make sure you don't pick a [GATT Service UUID](https://www.bluetooth.com/specifications/gatt/services/),

Reviewer: That format is markdown - this page is written in RST - see contributing guide for creating links with RST.

> Make sure you don't pick a .. _GATT Service UUID: https://www.bluetooth.com/specifications/gatt/services/,

Reviewer: ```suggestion
Make sure you don't pick a `GATT Service UUID <https://www.bluetooth.com/specifications/gatt/services/>`__,
```
